libs/blocks/marketo/marketo.js

```diff
@@ -91,6 +92,35 @@ export const setPreferences = (formData) => {
   Object.entries(formData).forEach(([key, value]) => setPreference(key, value));
 };
 
+const showSuccessSection = (formData, scroll = true) => {
+  try {
+    const show = (el) => {
+      el.classList.remove('hide-block');
+      if (scroll) el.scrollIntoView({ behavior: 'smooth' });
+    };
+    const section = formData[SUCCESS_SECTION].toLowerCase().replaceAll(' ', '-');
+    const success = document.querySelector(`.section.${section}`);
+    if (success) {
+      show(success);
+      return;
```

Comment:
Super nit: maybe rename section to successClass and success to section, might read a bit easier: 
```suggestion
    const successClass = formData[SUCCESS_SECTION].toLowerCase().replaceAll(' ', '-');
    
    const section = document.querySelector(`.section.${successClass}`);
    if (section) {
      show(section);
      return;
```